### PR TITLE
Prevent jump when clicking on action in menu.

### LINKED ISSUE #7367: Clicking on menu actions in the notebook makes it jump to the beginning
Fox example, if you click on `Insert > Insert Cell Above` or `View > Toggle Header`, the notebook jumps immediately to the beginning. It seems the same happens with all actions (although I haven't tried them all).

I'm using 3.0.0-dev, revision 4b40dd4

**Edit**: I tried it in Firefox 33 and Chrome 34

### PR DESCRIPTION
**Pull request by Carreau**
To do so prevent even default action;

Closes #7367

### CODE REVIEW

**Comment by mathieu1** [comment]
Don't forget the actions in `kernelselector.js` (new notebook and change kernel)

**Comment by Carreau** [comment]
> Don't forget the actions in kernelselector.js (new notebook and change kernel)

Indeed, commit on its way.

**Comment by Carreau** [comment]
And rebased.

File: IPython/html/static/notebook/js/kernelselector.js [hunk 1]
```diff
@@ -60,7 +60,10 @@ define([
             ks = this.kernelspecs[keys[i]];
             ks_submenu_entry = $("<li>").attr("id", "kernel-submenu-"+ks.name).append($('<a>')
                 .attr('href', '#')
-                .click($.proxy(this.set_kernel, this, ks.name))
+                .click(function(){
+                        $.proxy(this.set_kernel, this, ks.name)
+                        event.preventDefault();
```
Comment (minrk) [inline IPython/html/static/notebook/js/kernelselector.js]: shouldn't this be `return false`?

Comment (Carreau) [inline IPython/html/static/notebook/js/kernelselector.js]: Nop, otherwise menu does not get dismissed.

Comment (minrk) [inline IPython/html/static/notebook/js/kernelselector.js]: gotcha, that's the other half of `return false`. AOK, then.

**Pull request merged**